src/state/app.ts

```diff
@@ -122,6 +128,10 @@ export const defaultAppState: AppState = {
   currentLayerGroup: defaultLayerGroup,
   infoDialogs: {},
   userPostalCode: null,
+  intro: {
```

Comment:
What about the discussion in Slack? Should we add  `intro` to blacklist or not?
Otherwise we could probably merge this one?
@DanielHabenicht @ChristophWersal @kommander

Reply:
My 2 cents: it does not hurt to persist it in the local storage and we might need it later => I would merge it as it is.